pkg/controller/restore-operator/http.go

```diff
@@ -71,14 +71,20 @@ func (r *Restore) serveBackup(w http.ResponseWriter, req *http.Request) error {
 		return fmt.Errorf("no restore CR found for restore-name (%v)", restoreName)
 	}
 
-	logrus.Infof("serving backup for restore CR %v", restoreName)
+	logrus.Infof("serving backup for restore-name (%v)", restoreName)
```

Comment:
restore-name -> restore CR

:)

Reply:
hmm, we have mixed `restore-name` and `restore CR` within the code base. which one to use then?

Reply:
It's for different contexts.
For this context, please use `restore CR`.

Reply:
alright then